spec/Body/Body.vspec

```diff
@@ -51,6 +52,12 @@ Trunk.IsLocked:
   type: actuator
   description: Is trunk locked or unlocked. True = Locked. False = Unlocked.
 
+Trunk.IsLightOn:
+  datatype: boolean
+  type: actuator
+  description: Is trunk light on
+  comment: V3.0 Moved from Vehicle.Cabin.Lights.IsTrunkOn
```

Comment:
Here I see two options. Either we plan this PR for the minor release planned for end of February (v3.1), or for v4.0 planned for end of April. If we go for v4.0 we do not need to care about deprecation-comments, we just need to make sure it is mentioned in the CHANGELOG file. I think v4.0 is the best approach, as we otherwise for v3.1 would need to have both `Light` and `Lights` branch present.

(If we go for the minor release the comment shall say v3.1, as that is the first release where it will be deprecated.  This might be a side effect of using an old base branch, compare e.g. your [VERSION file](https://github.com/jdacoello/vehicle_signal_specification/blob/interior-lights/VERSION) with the latest [VERSION file](https://github.com/COVESA/vehicle_signal_specification/blob/master/VERSION)

Reply:
I am Ok with any of them. 

* If it is the minor release, I can also name it `Lights` instead, to be compatible with the existing branches. And then I can create a new issue to address the Singular names in branches.
* If it is the major release, I think there are some other aspects about the branches that can be considered. So far, I see that most of the effort is centred on the leaves of the tree. However, less attention is paid to the branches and the criteria by which the tree is categorising the information.

Reply:
@erikbosch FYI, I will adapt the PR to target the major release v.4.0. If the proposal is accepted, it should be merged in March.

Reply:
I think that makes sense, then we do not need the "deprecated" stuff. Also good from a timing perspective, as we have planned VSS 3.1 for end of February I would more or less like to have code freeze this week, and I believe this one needs to saty open until at least next week.